### DISCUSSION TITLE
why is it not working for me, i even tried it in a sandbox vault  but this plugin is not working for me

### DISCUSSION THREAD

**Discussion by ashuotaku** [natefrisch01/Graph-Link-Types#27] [General]
I have tried it many times but this plugin doesn't work for me 😭

**Comment by natefrisch01** [comment]
I am happy to help, if you send a screen recording or something that I can use to help see what's going wrong.

Using the sandbox vault might make debugging easier.

**Reply by ashuotaku** [reply]
Sorry for late reply as I was busy lately due to my exams

It started working when i setup a vault from zero, means it is working in my new vault, **although it only works when one instance of graph view is opened** when two instances of graph view is opened (either local or global) then **it only works in one graph view**.

**Reply by natefrisch01** [reply]
This is intended behavior - perhaps in the future we will add capabilities for multiples to be supported at the same time.

**Reply by ashuotaku** [reply]
ohh okay, thank you for your reply

**Comment by KuiyueRO** [comment]
downgrade version to 0.2.4

**Reply by natefrisch01** [reply]
May I ask what problems you are having with 3.x?

**Reply by KuiyueRO** [reply]
- 3.x didn't work with large vault(6k+)
- Obsidian freezes when I open global graph

**Reply by KuiyueRO** [reply]
- Dragging the graph is much more stuck than in 2.x

**Reply by natefrisch01** [reply]
I see. Thanks for the info! I think some of that I can try to fix in a performance release.

**Reply by KuiyueRO** [reply]
I suspect something has been querying, but I didn't receive any error message
